opentelemetry/proto/common/v1/common.proto

```diff
@@ -21,6 +21,36 @@ option java_package = "io.opentelemetry.proto.common.v1";
 option java_outer_classname = "CommonProto";
 option go_package = "go.opentelemetry.io/proto/otlp/common/v1";
 
+// inlined parquet-gen options
+import "google/protobuf/descriptor.proto";
+
+message MessageOptions {
```

Comment:
Chatted with @atoulme offline and thought that since whether a field is encoded as JSON or not is relatively generic it could be put in this file, for potential use in other signal-specific protos too.

But I hadn't looked well at the proto file, there is more than I was expecting and this table_name especially does seem quite parquet specific. @atoulme do you intend to use the features other than `is_json`? If not we should scope down to just that

Reply:
Definitely

Reply:
@anuraaga ptal should be good now.